content/applications/general/digest_emails.rst

```diff
@@ -0,0 +1,116 @@
+:show-content:
```

Comment:
This metadata is used to make the page accessible when it contains a `toctree` directive.
```suggestion
```